src/umbraco.cms/businesslogic/propertytype/PropertyTypeGroup.cs

```diff
@@ -44,19 +41,25 @@ public IEnumerable<PropertyType> GetPropertyTypes()
             return PropertyType.GetPropertyTypesByGroup(Id);
         }
 
-        //TODO: Verify support for master doctypes / mixins!
+        // note: this is used to delete all groups that inherit from a group, when the group is deleted,
+        // see the Delete method in this class - but it is all done in an obsolete way which does not
+        // take compositions in account + we delete the group but do not re-allocate the properties, etc.
+        // ALL THIS should be either removed, or refactored to use the new APIs - so... returning nothing
+        // from now on, which is just another way of being broken.
         public IEnumerable<PropertyTypeGroup> GetPropertyTypeGroups()
         {
-            var ptgs = new List<PropertyTypeGroup>();
-            using (var dr = SqlHelper.ExecuteReader(@"SELECT id FROM cmsPropertyTypeGroup WHERE parentGroupId = @id", SqlHelper.CreateParameter("@id", Id)))
-            {
-                while (dr.Read())
-                {
-                    ptgs.Add(GetPropertyTypeGroup(dr.GetInt("id")));
-                }
-            }
-
-            return ptgs;
+            return Enumerable.Empty<PropertyTypeGroup>();
+
+            //var ptgs = new List<PropertyTypeGroup>();

```

Comment:
I've merged this branch locally and already fixed this up, so no need for this one any more!